website/docs/getting-started/setup/instance-configuration/README.mdx

```diff
@@ -0,0 +1,146 @@
+---
+description: >-
+  Appsmith ships with third-party services & configurations that improve the app
+  building experience. All services & configurations are entirely optional.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Instance Configuration
+
+Configuration of your Appsmith instance allows you to tailor it to your specific needs. This process includes setting up elements like environment variables, authentication, emails, SSL, and custom domain. Proper configuration ensures that your instance is running efficiently. 
+
+## Configure Docker installations
+
+To configure a docker installation, simply open the folder of your installation and edit the **`stacks/configuration/docker.env`** file with the environment variables for the service.
+
+Remove # before the variables to ensure they're not commented
+
+```bash
+// Example variable configuration format
```

Comment:
Can we remove these comments from the code blocks. On the rendered documentation, these code blocks show up with a `copy` button. It is expected that people can just hit that button to copy the code, and paste in their terminal.

Having this comment _inside_ the code block means that they also copy this comment. So, can we either move this comment out into the previous sentence, or just remove it please?